Submitted By: Jim Gifford <jim@linuxfromscratch.org>
Date: 2006-09-16
Upstream Status: Unknown
Origin: Jim Gifford
Description: Change /lib/udev to /lib64/udev

diff -Naur udev-096.orig/extras/ata_id/Makefile udev-096/extras/ata_id/Makefile
--- udev-096.orig/extras/ata_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/ata_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/cdrom_id/Makefile udev-096/extras/cdrom_id/Makefile
--- udev-096.orig/extras/cdrom_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/cdrom_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/dasd_id/Makefile udev-096/extras/dasd_id/Makefile
--- udev-096.orig/extras/dasd_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/dasd_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/edd_id/Makefile udev-096/extras/edd_id/Makefile
--- udev-096.orig/extras/edd_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/edd_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/firmware/firmware.sh udev-096/extras/firmware/firmware.sh
--- udev-096.orig/extras/firmware/firmware.sh	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/firmware/firmware.sh	2006-09-16 14:41:40.000000000 -0700
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-FIRMWARE_DIRS="/lib/firmware /usr/local/lib/firmware"
+FIRMWARE_DIRS="/lib64/firmware /usr/local/lib64/firmware"
 
 err() {
     echo "$@" >&2
diff -Naur udev-096.orig/extras/firmware/Makefile udev-096/extras/firmware/Makefile
--- udev-096.orig/extras/firmware/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/firmware/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -13,7 +13,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/floppy/Makefile udev-096/extras/floppy/Makefile
--- udev-096.orig/extras/floppy/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/floppy/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/path_id/Makefile udev-096/extras/path_id/Makefile
--- udev-096.orig/extras/path_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/path_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -13,7 +13,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/run_directory/Makefile udev-096/extras/run_directory/Makefile
--- udev-096.orig/extras/run_directory/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/run_directory/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -15,7 +15,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/scsi_id/Makefile udev-096/extras/scsi_id/Makefile
--- udev-096.orig/extras/scsi_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/scsi_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -19,7 +19,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/usb_id/Makefile udev-096/extras/usb_id/Makefile
--- udev-096.orig/extras/usb_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/usb_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -16,7 +16,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/extras/volume_id/lib/Makefile udev-096/extras/volume_id/lib/Makefile
--- udev-096.orig/extras/volume_id/lib/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/volume_id/lib/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -5,8 +5,8 @@
 # Released under the GNU General Public License, version 2.
 #
 includedir = 	${prefix}/usr/include
-libdir =	${prefix}/lib
-usrlibdir =	${prefix}/usr/lib
+libdir =	${prefix}/lib64
+usrlibdir =	${prefix}/usr/lib64
 
 INSTALL = /usr/bin/install -c
 INSTALL_DATA  = ${INSTALL} -m 644
diff -Naur udev-096.orig/extras/volume_id/Makefile udev-096/extras/volume_id/Makefile
--- udev-096.orig/extras/volume_id/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/extras/volume_id/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -14,7 +14,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev/
 
diff -Naur udev-096.orig/Makefile udev-096/Makefile
--- udev-096.orig/Makefile	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/Makefile	2006-09-16 14:40:50.000000000 -0700
@@ -98,7 +98,7 @@
 sbindir =	${prefix}/sbin
 usrbindir =	${prefix}/usr/bin
 usrsbindir =	${prefix}/usr/sbin
-libudevdir =	${prefix}/lib/udev
+libudevdir =	${prefix}/lib64/udev
 mandir =	${prefix}/usr/share/man
 configdir =	${etcdir}/udev
 udevdir =	/dev
diff -Naur udev-096.orig/udev_rules_parse.c udev-096/udev_rules_parse.c
--- udev-096.orig/udev_rules_parse.c	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/udev_rules_parse.c	2006-09-16 14:40:50.000000000 -0700
@@ -396,9 +396,9 @@
 				if (pos)
 					pos[0] = '\0';
 
-				/* allow programs in /lib/udev called without the path */
+				/* allow programs in /lib64/udev called without the path */
 				if (strchr(file, '/') == NULL) {
-					strlcpy(file, "/lib/udev/", sizeof(file));
+					strlcpy(file, "/lib64/udev/", sizeof(file));
 					strlcat(file, value, sizeof(file));
 					pos = strchr(file, ' ');
 					if (pos)
diff -Naur udev-096.orig/udev_utils_run.c udev-096/udev_utils_run.c
--- udev-096.orig/udev_utils_run.c	2006-07-09 12:48:19.000000000 -0700
+++ udev-096/udev_utils_run.c	2006-09-16 14:40:50.000000000 -0700
@@ -124,9 +124,9 @@
 		}
 	}
 
-	/* allow programs in /lib/udev called without the path */
+	/* allow programs in /lib64/udev called without the path */
 	if (strchr(argv[0], '/') == NULL) {
-		strlcpy(program, "/lib/udev/", sizeof(program));
+		strlcpy(program, "/lib64/udev/", sizeof(program));
 		strlcat(program, argv[0], sizeof(program));
 		argv[0] = program;
 	}
